frontend/Auth/FinishResetPassword.tsx

```diff
@@ -63,60 +63,108 @@ export function FinishResetPassword(props: RouterProps) {
+      tokenValidationResponse?.tokenIsExpired === false ? (
+        <form
+          onSubmit={unary(submitMutation.mutate)}
+          className="space-y-32 relative lg:max-w-sm w-64"
+        >
+          <div>
+            <h1 className="text-gray-500 place-self-start mb-2 text-5xl lg:text-xl">
+              Set New Password
+            </h1>
+
+            <p className="text-left text-gray-600 py-8 text-4xl lg:text-sm">
+              Enter your new password below.
+            </p>
+
+            <FormField className="mb-4">
+              <FormFieldLabel
+                className="text-3xl lg:text-xs"
+                htmlFor="username"
+              >
+                Email
+              </FormFieldLabel>
+              <Input
+                id="username"
+                type="email"
+                placeholder={userEmail}
+                value={userEmail}
+                disabled
+                className="text-gray-500"
+              />
+            </FormField>
+
+            <FormField className="mb-4">
+              <FormFieldLabel
+                className="text-3xl lg:text-xs"
+                htmlFor="password"
+              >
+                Password
+              </FormFieldLabel>
+              <Input
+                id="password"
+                className="text-5xl lg:text-sm"
+                type="password"
+                value={finishResetPasswordFormData.password}
+                onChange={(evt) =>
+                  setFinishResetPasswordFormData({
+                    ...finishResetPasswordFormData,
+                    password: evt.target.value,
+                  })
+                }
+                required
+              />
+            </FormField>
+          </div>
+
+          <div className="inset-x-0 my-8 bottom-0">
+            <Button
+              kind={ButtonKind.primary}
+              type="submit"
+              className="w-full h-24 lg:h-10 text-3xl lg:text-sm"
+            >
+              Submit New Password
+            </Button>
+          </div>
+        </form>
+      ) : (
+        <div className="space-y-32 relative lg:max-w-sm w-64">
+          <div>
+            <h1 className="text-gray-500 place-self-start mb-2 text-5xl lg:text-xl">
+              Set New Password
+            </h1>
+
+            {tokenValidationResponse?.tokenIsValid == false ? (
+              <p className="text-left text-gray-600 py-8 text-4xl lg:text-sm">
+                This reset password link is not valid. Please try{" "}
+                <a href="/reset-password" style={{ color: "1c69f9" }}>
+                  resetting your password
+                </a>{" "}
+                again.
+              </p>
+            ) : (
+              <p className="text-left text-gray-600 py-8 text-4xl lg:text-sm">
+                This reset password link is expired. Please try{" "}
+                <a href="/reset-password" style={{ color: "1c69f9" }}>
```

Comment:
By default, `<a>` causes a page reload, but `<Link>` does not. It's generally preferable to avoid page reloads when possible (in the context of single page applications), because it's more performant that way.

So let's switch to react-router's `<Link>` component.

Also, it's unusual to set a custom color here via style. Adding the new color to the tailwind config is preferred, so that you can use css classes to set the color. Also, I'm not sure you can use hex color codes without prefixing them with `#`. See https://github.com/JustUtahCoders/flax-crm/blob/20925dcfa11a10cb0a1e9340acfd49a9683a5d8f/tailwind.config.cjs#L5 for where you can add a new color - let's call it `classic-link`

```suggestion
                <Link to="/reset-password" className="text-classic-link">
```

Reply:
Thank you for explaining this!  So helpful.